RecoLocalCalo/EcalRecProducers/python/ecalMultiFitUncalibRecHitGPUcfi.py

```diff
@@ -0,0 +1,86 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
File name is missing an underscore, should be `ecalMultiFitUncalibRecHitGPU_cfi`.

Reply:
Should be addressed by https://github.com/cms-patatrack/cmssw/pull/563 (still in progress)